website/source/docs/builders/qemu.html.md.erb

```diff
@@ -319,6 +320,13 @@ will bind to their own SSH port as determined by each process. This will also
 work with WinRM, just change the port forward in `qemuargs` to map to WinRM's
 default port of `5985` or whatever value you have the service set to listen on.
 
+-   `use_backing_file` (boolean) - Only applicable when `disk_image` is `true`
+    and `format` is `qcow2`, set this option to `true` to create a new QCOW2
+    file that uses the file located at `iso_url` as a backing file. The new file
+    will only contain blocks that have changed compared to the backing file, so
+    enabling this option can significantly reduce disk usage. This also faster
```

Comment:
@SwampDragons I ran a test on a VM with slow backing storage and it didn't save much time. Should I remove `This also faster because it does not create a complete copy of the source.`?